web/src/config.ts

```diff
@@ -1,7 +1,6 @@
 import chairmanProfile from "./assets/profiles/chairman.jpg"
 import financialManagerProfile from "./assets/profiles/financialManager.jpg"
-import eventsProfile from "./assets/profiles/events.jpg"
-import logisticsProfile from "./assets/profiles/logistics.jpg"
```

Comment:
Would prefer to keep this commented out so its' easier for future DAG to add a future photo

Reply:
That sounds fair, I'll add it back in. Eventually I want to improve the documentation for the site to make it clearer.